modules/ROOT/pages/single-sign-on-with-kerberos.adoc

```diff
@@ -289,13 +289,13 @@ If you want Bonita engine to create the accounts on the fly when a user accessin
 NOTE: Activating this option means any user logged in on the domain trying to access Bonita will have an account created automatically in Bonita Database.
 
 . If your Domain Controller is correctly configured, you are done. +
-Then you can start the bundle and try to access a portal page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are automatically logged in.
+Then you can start the bundle and try to access any Bonita Application page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are automatically logged in.
 
 Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `kerberos.auth.standard.allowed` to true or define a whitelist with the property `auth.tenant.standard.whitelist`) to be able to log in without an account on AD.
```

Comment:
on AD => on Active Directory.

Reply:
not everyone knows what AD is.

Reply:
True, but it's just that there are a lot of references to AD in this entire file. We could mention that AD stands for Active directory at the top of the file (when we first mention Active directory) or replace all occurrences of AD with Active Directory. Your choice

Reply:
deal!